doc/Command-Reference.md

```diff
@@ -1730,6 +1734,145 @@ This command is used to remove particular IPv4 or IPv6 BGP neighbor configuratio
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#bgp)
 
+## Container Auto-restart
+SONiC introduced a feature about auto-restarting docker containers
+if one of critical processes accidently crashed or exited. The motivation behind
+this feature is that if one critical process in a docker container crashed, then
+the whole container will not work correctly and we can try to restart to make 
+it enter into the healthy state.
+
+### Container Auto-restart show commands
+
+**show container feature autorestart**
+
+This command will display the status of auto-restart feature for containers.
+
+- Usage:
+  ```
+  show container feature autorestart [<container_name>]
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ show container
+  Usage: show container [OPTIONS] COMMAND [ARGS]...
+
+  Show container
+
+  Options:
+    -?, -h, --help  Show this message and exit.
+
+  Commands:
+    feature  Show container feature
+  ```
+
+  ```
+  admin@sonic:~$ show container feature
+  Usage: show container feature [OPTIONS] COMMAND [ARGS]...
+
+  Show container feature
+
+  Options:
+    -?, -h, --help  Show this message and exit.
+
+  Commands:
+    autorestart  Show whether the auto-restart feature for container(s) is
+                 enabled or disabled
+  ```
+
+  ```
+  admin@sonic:~$ show container feature autorestart
+  Container Name    Status
+  --------------    --------
+  database          enabled
+  syncd             enabled
+  teamd             disabled
+  dhcp_relay        enabled
+  lldp              enabled
+  pmon              enabled
+  bgp               enabled
+  swss              disabled
+  telemetry         enabled
+  sflow             enabled
+  snmp              enabled
+  radv              disabled
+  ```
+
+Optionally, you can specify a container name in order to display the auto-restart
+feature status for that container only.
+
+- Usage:
+  ```
+  show container feature autorestart database
+  ```
```

Comment:
No need to repeat the usage here, as it is above. Please remove.

Reply:
Removed.